static/src/javascripts/projects/commercial/modules/header-bidding/types.d.ts

```diff
@@ -90,7 +94,8 @@ type BidderCode =
 	| 'sonobi'
 	| 'triplelift'
 	| 'trustx'
-	| 'xhb';
+	| 'xhb'
+	| 'criteo';
```

Comment:
Could we keep these ordered alphabetically?

There’s a `sort lines` option in VSCode.

Reply:
TIl `sort lines`!